index.js

```diff
@@ -52,7 +52,7 @@ function ensureAllCommentsPrinted(astComments) {
 function format(text, opts, addAlignmentSize) {
   addAlignmentSize = addAlignmentSize || 0;
 
-  const ast = parser.parse(text, opts);
+  const ast = opts.astTransform(parser.parse(text, opts));
```

Comment:
you'll likely want to pass other options in the future like filename, rawText..., that's good that you have more than one argument :)